scripts/geodata/polygons/index.py

```diff
@@ -161,7 +161,7 @@ def to_polygon(cls, coords, holes=None, test_point=None):
             return None
 
         # Fix for polygons crossing the 180th meridian
-        lons = [lon for lon, lat in coords]
+        lons = [coordinate[0] for coordinate in coords]
```

Comment:
Some coordinates in the latest data are in the form of (latitude, longitude, altitude)